autoload/lookup.vim

```diff
@@ -44,7 +44,7 @@ endfunction
 
 " s:find_local_func_def() {{{1
 function! s:find_local_func_def(funcname) abort
-  if search('\c\v<fu%[nction]!?\s+%(s:|\<sid\>)\zs\V'.a:funcname.'\>', 'bsw') != 0
+  if search('\c\v<fu%[nction]!?\s+%(s:|\<sid\>|\<SID\>)\zs\V'.a:funcname.'\>', 'bsw') != 0
```

Comment:
I think maybe we should not use `\c`, here only `sid` is ignorecase, `fu` `function` and `s:` is not ignorecase.